### PR TITLE
Kinder, gentler death

### PR DESCRIPTION
**Pull request by SabreCat**
Revises the "death" modal to be less rage-inducing, removing references to dying and giving the user more encouragement. Displays the user's avatar in its "sleepy" look, for maximum personality!

### CODE REVIEW

File: website/public/css/shared.styl [hunk 1]
```diff
@@ -60,3 +60,7 @@ ul
   .tasks-lists .row .col-md-3
     padding-right: 0
     padding-left: 0
+
+.flex-column
+  display: flex
```
Comment (crookedneighbor) [inline website/public/css/shared.styl:65]: Do we care about browsers that don't support flex?

File: website/views/shared/modals/death.jade [hunk 2]
```diff
@@ -1,16 +1,40 @@
...
+    .container-fluid(ng-controller='UserCtrl')
+      .col-md-6
+        .hero-stats
+          .meter-label(tooltip=env.t('health'))
+            span.glyphicon.glyphicon-heart
+          .meter.health(tooltip='{{Math.round(user.stats.hp * 100) / 100}}')
+            .bar(ng-style='{"width": Shared.percent(user.stats.hp, Shared.maxHealth)+"%"}')
+            span.meter-text.value
+              | {{Math.ceil(user.stats.hp)}} / {{::Shared.maxHealth}}
+        figure.herobox.text-center
```
Comment (crookedneighbor) [inline website/views/shared/modals/death.jade:16]: This is essentially herobox sans background, right? Anyway we can DRY this up?

**Comment by Alys** [comment]
A screenshot for posterity and so no one else has to kill an innocent avatar to see this. :)

screen shot 2015-07-25 at 9 18 49 am

File: website/public/css/shared.styl (hunk 1 above)
Comment (SabreCat) [inline website/public/css/shared.styl:65]: Oh, good question. What are those? If it's just old IE, we don't care, but
if it doesn't work on something current, that's a problem.

File: website/views/shared/modals/death.jade (hunk 2 above)
Comment (SabreCat) [inline website/views/shared/modals/death.jade:16]: Hero box minus background and with the skin forced to "sleeping". I could
probably figure a way to modularize it, aye!

File: website/public/css/shared.styl (hunk 1 above)
Comment (crookedneighbor) [inline website/public/css/shared.styl:65]: http://caniuse.com/#feat=flexbox

**Pull request merged**